CONTRIBUTING.md

```diff
@@ -29,6 +29,22 @@ And for another example, here are commits to add support for `pytest-benchmark`:
 - Add workflows for test and example: https://github.com/rhysd/github-action-benchmark/commit/1e4ebf2e9ecde9e7620661c60455b22837a2bdaf
 - Add documentation: https://github.com/rhysd/github-action-benchmark/commit/895f92f564521597492bd281cbf6c8efd39f628e
 
+## Running CI workflow on a forked repo
+
+Since the benchmark data includes the URL of the repository the tests and examples will fail when the repo is forked. In order to get the ci workflow and all examples running the URL has to be changed
+
+### Change workflows and tests
+
+1. in [`ci_validate_modification.ts`](scripts/ci_validate_modification.ts) in the `validateJSON` function replace `https://github.com/rhysd/github-action-benchmark` with your repo URL (i.e. `https://github.com/<YOU>/github-action-benchmark`)
+2. in all workflow (`.yml`) in `.github/workflows` replace `alert-comment-cc-users: '@rhysd'` with your user
```

Comment:
This is better in terms of maintenance since I'm sometimes not a user of the benchmarking tool. Thank you for adding this.